scripts/setup/linux/ubuntu/xenial/install_prereqs

```diff
@@ -36,90 +36,23 @@ if [[ "${EUID}" -ne 0 ]]; then
   exit 1
 fi
 
-apt update -qq
-apt install -qq --no-install-recommends $(tr '\n' ' ' <<EOF
-curl
-software-properties-common
-EOF
-)
-
-add-apt-repository -s 'deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8'
-curl -L https://bazel.build/bazel-release.pub.gpg | apt-key add -
-
-add-apt-repository ppa:dreal/dreal
+# NOTE: Assumes that the drake (binary) install_prereqs script will also be run.  Only install what we need to download drake and any additional requirements for this package.
 
 apt update -qq
 
 # Install development and Drake dependencies.
 apt install -qq --no-install-recommends $(tr '\n' ' ' <<EOF
-bazel
-bison
-ca-certificates
-clang-format-4.0
-cmake
-coinor-libclp-dev
-coinor-libipopt-dev
-flex
-g++
-git
-libboost-all-dev
-libbz2-dev
-libccd-dev
-libexpat1-dev
-libfreetype6-dev
-libgflags-dev
-libglib2.0-dev
-libglu1-mesa-dev
-libhdf5-dev
-libibex-dev
-libjpeg8-dev
-libjsoncpp-dev
-liblapack-dev
-liblz4-dev
-libnetcdf-cxx-legacy-dev
-libnetcdf-dev
-libnlopt-dev
-libogg-dev
-libpng12-dev
-libprotobuf-dev
-libqt5opengl5-dev
-libqt5x11extras5-dev
-libtheora-dev
-libtiff5-dev
-libtinyxml-dev
-libtinyxml2-dev
-libtool
-libxml2-dev
-libxt-dev
-libyaml-cpp-dev
-make
-openjdk-8-jdk
-openssh-client
-patchelf
-pkg-config
-protobuf-compiler
-python-dev
-python-gtk2
-python-lxml
+apt-transport-https
+curl
+python
```

Comment:
See https://github.com/RobotLocomotion/drake/pull/7961. You can just remove this now and nothing should break given it is implicitly installed.

Reply:
currently I use python to parse the json to know which url to download drake from.  ;-)

(the textbook install_prereqs actually runs before the drake install_prereqs on CI, and because of that and the need for e.g. curl, I have the order the same in the textbook install instructions)

Reply:
Ok, that makes sense.